StudioCore/ParamEditor/ParamBank.cs

```diff
@@ -812,7 +815,10 @@ public static void LoadAuxBank(string path, string looseDir, string enemyPath)
             }
             else if (locator.Type == GameType.DarkSoulsIISOTFS)
             {
-                newBank.LoadParamsDS2FromFile(new List<string>{looseDir}, path, enemyPath);
+                newBank.LoadParamsDS2FromFile(new List<string>
```

Comment:
Please watch out for autoformatting, while usually benign it can end up making some odd choices, plus it clutters diffs